### PR TITLE
#138 - Extend integration tests in ci

### PR DESCRIPTION
**Pull request by michael-82**
- add and set up blaze and keycloak
- execute a real query against blaze and check if a result > 0 is returned

### CODE REVIEW

**Review by EmteZogaf — CHANGES_REQUESTED** [review]
For testing that a query is sent and the result is received a single test patient with the same gender as used in the test query should be sufficient. No need to add that many test data.

**Comment by juliangruendner** [comment]
> For testing that a query is sent and the result is received a single test patient with the same gender as used in the test query should be sufficient. No need to add that many test data.

i agree

**Review by juliangruendner — CHANGES_REQUESTED** [review]
less testdate should be sufficient - see comment from @EmteZogaf

File: .github/scripts/create-keycloak-user.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -e
+
+docker exec -u0 integration-test_auth_1 /opt/keycloak/bin/kcadm.sh config credentials --server http://localhost:8080/auth --realm master --user keycloakadmin --password keycloak
+docker exec -u0 integration-test_auth_1 /opt/keycloak/bin/kcadm.sh create users -s username=testuser -s email=test@example.com -s enabled=true -s emailVerified=true -r feasibility
+docker exec -u0 integration-test_auth_1 /opt/keycloak/bin/kcadm.sh add-roles --uusername testuser --rolename FeasibilityUser -r feasibility
```
[the remark this answers is not in the document]
Comment (michael-82) [reply .github/scripts/create-keycloak-user.sh:5]: That is indeed the correct way. See https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.3/html/server_administration_guide/the_admin_cli#user_operations "Creating a user" and "Adding realm roles to a user"

**Comment by michael-82** [comment]
Changed according to request(s)

**Review by EmteZogaf — APPROVED** [review]
lgtm

**Pull request merged**